presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -126,4 +126,34 @@ There is also an additional parameter:
     If the time period is less than or equal to the parameter value, the request
     is valid.
     If the time period exceeds the parameter value, the request is rejected as
-    authentication failure (HTTP 401).
\ No newline at end of file
+    authentication failure (HTTP 401).
+
+Async Data Cache and Prefetching
+--------------------------------
+
+``num-connector-io-threads``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``integer``
+    * **Default value:** ``30``
+
+    Size of IO executor for connectors to do preload/prefetch.  Prefetch is
+    disabled if ``num-connector-io-threads`` is set to zero.
+
+``async-data-cache-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``bool``
+    * **Default value:** ``true``
+
+    Whether async data cache is enabled.  Setting ``async-data-cache-enabled``
+    to ``false`` disables split prefetching in table scan.
+
+``async-cache-ssd-gb``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``integer``
+    * **Default value:** ``0``
```

Comment:
Are we missing some other configs re: SSD. I assume one needs to specify rott directory or something, no?

Reply:
Just you will need to set SSD size in another property to indicate how much space is available